setup.py

```diff
@@ -68,7 +90,10 @@
     packages=find_packages(include=['neuraxle*']),
     test_suite="testing",
     setup_requires=["pytest-runner"],
-    install_requires=['numpy', 'scipy', 'matplotlib', 'scikit-learn>=0.20.3', 'joblib>=0.13.2'],  # , 'tensorflow'],
+    install_requires=[
+        'numpy', 'scipy', 'matplotlib', 'scikit-learn>=0.20.3', 'joblib>=0.13.2',
+        'Flask>=1.1.1', 'Flask-RESTful>=0.3.7'
+    ],
```

Comment:
Why isn't this already merged in the dev branch? well I didn't expect to see those changes here. Anyway it seems fine.